src/py/flwr/cli/ls.py

```diff
@@ -255,19 +257,66 @@ def _to_table(run_list: list[tuple[str, ...]]) -> Table:
     return table
 
 
+def _to_json(run_list: list[tuple[str, ...]]) -> str:
+    """Format run status list to a JSON formatted string."""
+
+    def _remove_bbcode_tags(strings: tuple[str, ...]) -> tuple[str, ...]:
```

Comment:
Just a silly question: do we add bbcode tags somewhere? Does the `_format_runs` add these tags?

Reply:
Yes, it's the `status_style`. 😅

Reply:
This looks like a useful function. Let's move it to `cli/utils.py`?